src/build/__main__.py

```diff
@@ -250,9 +251,7 @@ def main_parser() -> argparse.ArgumentParser:
     """
     Construct the main parser.
     """
-    # mypy does not recognize module.__path__
-    # https://github.com/python/mypy/issues/1422
-    paths: Iterable[str] = build.__path__  # type: ignore
+    paths = build.__path__
```

Comment:
We don't need the assignment here anymore, we only had it so we could replace the type of `__path__`.

Reply:
Thanks!